src/layouts/Header/components/Navigator/Navigator.tsx

```diff
@@ -63,7 +64,16 @@ function Navigator({ currentUser }: NavigatorProps) {
     pageName = 'accountAdmin'
   }
 
-  return <MyContextSwitcher pageName={pageName} />
+  return (
+    <div className="flex items-center">
+      <MyContextSwitcher pageName={pageName} />
+      {isCurrentUserPartOfOrg ? null : (
+        <Label variant="plain" className="hidden sm:block">
```

Comment:
`className="hidden sm:block"` makes it so on very small screens the pill is hidden (instead of overlapping and looking ugly)